.github/workflows/build.yaml

```diff
@@ -369,35 +369,33 @@ jobs:
           retention-days: 1
 
   # See https://github.com/flatpak/flatpak-github-actionsfor examples
+  # It also contains instructions on how to compile for other architectures via QEMU
   flatpak:
     name: Flatpak
-    runs-on: ubuntu-latest
-    container:
-      image: bilelmoussaoui/flatpak-github-actions:gnome-45
-      options: --privileged
     strategy:
       matrix:
-        arch: [x86_64, aarch64]
+        variant:
+          - arch: x86_64
+            runner: ubuntu-latest
+          - arch: aarch64
+            runner: ARM64
       # Don't fail the whole workflow if one architecture fails
       fail-fast: false
+    runs-on: ${{ matrix.variant.runner }}
+    container:
+      # BUG https://github.com/flatpak/flatpak-github-actions/pull/179
+      # image: bilelmoussaoui/flatpak-github-actions:gnome-45
```

Comment:
is this still true?

Reply:
I would prefer the image from `flatpak` instead of `flathub-infra` as it is more general and smaller and we are also using the action from that repository but the maintainer is not very responsive at the moment. So I suggest keeping the comment for future reference.

Reply:
Interesting, I thought it was the other way round as this was based on the small `fedora` image. But it looks like it has a lot of inefficiencies like installing Flatpak builder twice (`dnf` and `flatpak`) for example.

@barthalion Which GitHub actions and Docker images do you recommend people to use? It is a little bit confusing with the many GitHub organizations and forks...

Reply:
@barthalion thanks. Are then just using `@master` for the version? Or pin a sha and update with dependabot?